server/weaverbird/backends/pandas_executor/steps/utils/cast.py

```diff
@@ -18,7 +21,10 @@ def cast_to_str(s: Series) -> Series:
 
 
 def cast_to_datetime(s: Series) -> Series:
-    return to_datetime(s, errors='coerce')  # cast errors will result in NaT values
+    timestamp_unit = 'ms' if s.dtype == 'int64' else None
```

Comment:
Not clear to me why only when the type is `int64`. What cases does this type represent and what other cases could happen?

Reply:
This type represent the "converting timestamp to dates" case,other case

Reply:
Gonna try being a little more verbose and will add comments if necessary